freqtrade/persistence/models.py

```diff
@@ -587,7 +587,7 @@ def calculate_interest(self, interest_rate: Optional[float] = None) -> Decimal:
             return zero
 
         open_date = self.open_date.replace(tzinfo=None)
-        now = datetime.utcnow()  # TODO-mg: Update to self.close_date
+        now = (self.close_date or datetime.utcnow()).replace(tzinfo=None)
```

Comment:
Please don't use `utcnow()` - but use `datetime.utc(timezone.utc)` instead.

`datetime.utcnow()` is fluky as hell (you can't get a correct timestamp off of it unless your system time is also UTC) - as it gets you the date in UTC - but without having the timezone set.

Reply:
I get this error `AttributeError: type object 'datetime.datetime' has no attribute 'utc'`
When I use `datetime.utc(timezone.utc)`

Reply:
yeah - it should be `datetime.now(timezone.utc)` ... sorry bout that